app/src/main/java/net/gsantner/markor/activity/SettingsActivity.java

```diff
@@ -376,9 +376,10 @@ public void onFsViewerConfig(FilesystemViewerData.Options dopt) {
                 case R.string.pref_key__plaintext__reorder_actions:
                 case R.string.pref_key__markdown__reorder_actions:
                 case R.string.pref_key__zimwiki__reorder_actions:
+                case R.string.pref_key__txt2tags__reorder_actions:
                 case R.string.pref_key__todotxt__reorder_actions: {
                     Intent intent = new Intent(getActivity(), ActionOrderActivity.class);
-                    intent.putExtra(ActionOrderActivity.EXTRA_FORMAT_KEY, (keyResId == R.string.pref_key__markdown__reorder_actions) ? R.id.action_format_markdown : (keyResId == R.string.pref_key__todotxt__reorder_actions ? R.id.action_format_todotxt : (keyResId == R.string.pref_key__zimwiki__reorder_actions ? R.id.action_format_zimwiki : R.id.action_format_plaintext)));
+                    intent.putExtra(ActionOrderActivity.EXTRA_FORMAT_KEY, (keyResId == R.string.pref_key__markdown__reorder_actions) ? R.id.action_format_markdown : (keyResId == R.string.pref_key__todotxt__reorder_actions ? R.id.action_format_todotxt : (keyResId == R.string.pref_key__zimwiki__reorder_actions ? R.id.action_format_zimwiki : (keyResId == R.string.pref_key__txt2tags__reorder_actions ? R.id.action_format_txt2tags : R.id.action_format_plaintext))));
```

Comment:
Can we replace this with a switch? A line this long is pretty unreadable

Reply:
yes, sure, I've just adapted the code to what was already there. I must admit I don't know much java, so I've looked how it was before.

Reply:
I know it's not pretty but keep it in the PR farvardin.

If it's just that line, preferrably we/I take care outside of the PR, after merge.

Reply:
> I know it's not pretty but keep it in the PR farvardin.

ok, I won't touch it.